src/generator/named-types.ts

```diff
@@ -197,7 +204,7 @@ function fetchRootTypes(module: Module): { valueType: ValueType; source: ValueTy
 
 function recursiveVisitMembersType(
   valueType: ValueType,
-  visit: (membersType: NamedType | TupleType | UnionType, path: string) => void,
+  visit: (membersType: NamedType | TupleType | UnionType | TypeUnion, path: string) => void,
```

Comment:
What the difference between `UnionType` and `TypeUnion`. The names are a bit confusing

Reply:
UnionType is 1 | 2
TypeUnion is boolean | string